gradle/libs.versions.toml

```diff
@@ -1,33 +1,34 @@
 # Sharing dependency versions between projects
 # https://docs.gradle.org/current/userguide/platforms.html
 [versions]
-aboutlibraries = "11.2.3"
-accompanist = "0.36.0"
-androidx-compose = "1.7.5"
+aboutlibraries = "11.4.0"
+accompanist = "0.37.0"
+androidx-compose = "1.7.7"
 androidx-benchmark="1.3.3"
-androidx-datastore = "1.1.1"
+androidx-datastore = "1.1.2"
+androidx-hilt = "1.2.0"
 androidx-lifecycle = "2.8.7"
-androidx-navigation = "2.8.4"
+androidx-navigation = "2.8.6"
 androidx-room = "2.6.1"
 androidx-test = "1.6.1"
 apollo = "4.1.0"
-chucker = "4.0.0"
-coil = "3.0.3"
-coroutines = "1.9.0"
-dagger = "2.52"
+chucker = "4.1.0"
+coil = "3.0.4"
+coroutines = "1.10.1"
+dagger = "2.55"
 detekt = "1.23.7"
 diktat = "1.2.5"
 gradle = "8.10.2"
```

Comment:
Why is this still `8.10.2`? Please change it to `8.12.1` and run the wrapper again, so that it will update also the `gradle/wrapper/gradle-wrapper.properties` automatically